backend/src/schema/resolvers/reports.js

```diff
@@ -153,14 +155,14 @@ export default {
       const { id } = parent
       let reviewed
       const readTxPromise = session.readTransaction(async tx => {
-        const allReportsTransactionResponse = await tx.run(
-          `
-            MATCH (resource)<-[:BELONGS_TO]-(report:Report {id: $id})<-[review:REVIEWED]-(moderator:User)
-            RETURN moderator, review
-            ORDER BY report.updatedAt DESC, review.updatedAt DESC
-          `,
-          { id },
-        )
+        const cypher = `
+          MATCH (resource)<-[:BELONGS_TO]-(report:Report {id: $id})<-[review:REVIEWED]-(moderator:User)
+          RETURN moderator, review
+          ORDER BY report.updatedAt DESC, review.updatedAt DESC
+        `
+        const params = { id }
+        const allReportsTransactionResponse = await tx.run(cypher, params)
+        log(allReportsTransactionResponse)
```

Comment:
same question as your other `PR` @roschaefer 
why so much refactoring just to add `log`??

Reply:
I want to see the database calls being made when I set `DEBUG=human-connection:neo4j:*`. So I implemented this helper function which outputs the cypher statement and/or the database call statistics, the counters. We should always call the `log` method from now on when we implement database calls.

Reply:
that part, I get... I don't understand how why that needed so much refactoring
I have refactored it and pushed it up.
Would this somehow not work with the code you added... 
It would be nice to know how to use as well
If I run `DEBUG=human-connection:neo4j:* yarn dev` it throws this error
```
No matches for wildcard “DEBUG=human-connection:neo4j:*”. See `help expand`.
env DEBUG=human-connection:neo4j:* yarn dev
```
so I'm assuming the wildcard needs to be replaced by something, but what? Ahh 
it takes either `human-connection:neo4j:cypher` or `human-connection:neo4j:stats`??

Reply:
On fish shell:
```fish
env DEBUG="human-connection:neo4j:*" yarn run dev
```
if you don't escape the `*` your shell believes it's a file glob pattern.

Reply:
great thanks @roschaefer 
then run a query and it will give me some crazy feedback?

Reply:
ahhh, ok here are the result of running a `reports` query
```
2019-12-10T16:04:20.346Z human-connection:neo4j:cypher 
            MATCH (report:Report)-[:BELONGS_TO]->(resource)
            WHERE (resource:User OR resource:Post OR resource:Comment)
            
            WITH report, resource,
            [(submitter:User)-[filed:FILED]->(report) |  filed {.*, submitter: properties(submitter)} ] as filed,
            [(moderator:User)-[reviewed:REVIEWED]->(report) |  reviewed {.*, moderator: properties(moderator)} ] as reviewed,
            [(resource)<-[:WROTE]-(author:User) | author {.*} ] as optionalAuthors,
            [(resource)-[:COMMENTS]->(post:Post) | post {.*} ] as optionalCommentedPosts,
            resource {.*, __typename: labels(resource)[0] } as resourceWithType
            WITH report, optionalAuthors, optionalCommentedPosts, reviewed, filed,
            resourceWithType {.*, post: optionalCommentedPosts[0], author: optionalAuthors[0] } as finalResource
            RETURN report {.*, resource: finalResource, filed: filed, reviewed: reviewed }
            ORDER BY report.createdAt DESC
             
          
2019-12-10T16:04:20.348Z human-connection:neo4j:cypher {}
2019-12-10T16:04:20.349Z human-connection:neo4j:stats StatementStatistics { _stats: { nodesCreated: 0, nodesDeleted: 0, relationshipsCreated: 0, relationshipsDeleted: 0, propertiesSet: 0, labelsAdded: 0, labelsRemoved: 0, indexesAdded: 0, indexesRemoved: 0, constraintsAdded: 0, constraintsRemoved: 0 } }
2019-12-10T16:04:20.349Z human-connection:neo4j:stats { resultConsumedAfter: 2, resultAvailableAfter: 56 }
```